src/main/java/com/google/sps/servlets/ChatServlet.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.pusher.rest.Pusher;
+import java.io.IOException;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.Collections;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/chat")
+public final class ChatServlet extends HttpServlet {
+
+  final private String APP_ID = "1024158";
+  final private String CLIENT_KEY = "d15fbbe1c77552dc5097";
+  final private String CLIENT_SECRET = "91fd789bf568ec43d2ee";
+  private Pusher pusher;
+  private Gson gson;
+
+  @Override
+  public void init() {
+    pusher = new Pusher(APP_ID, CLIENT_KEY, CLIENT_SECRET);
```

Comment:
This is fine for now, but as other parts of the code start needing a pusher client, it would be cool to have a helper class that does all this setup.

Reply:
What I am trying to say though, is don't worry about making any changes to this part of the code now. Once even one more use case comes up, that would be a good time to refactor it.